samples/nrf9160/multicell_location/prj.conf

```diff
@@ -34,9 +34,9 @@ CONFIG_MULTICELL_LOCATION_MAX_NEIGHBORS=10
 # CONFIG_MODEM_JWT=y
 
 # nRF Cloud location service configuration
-# CONFIG_MULTICELL_LOCATION_SERVICE_NRF_CLOUD=y
-# CONFIG_NRF_CLOUD_REST=y
-# CONFIG_MODEM_JWT=y
+CONFIG_MULTICELL_LOCATION_SERVICE_NRF_CLOUD=y
```

Comment:
What has changed that this would be needed now? Did they add a new test?
This sample has always had no service configuration so it couldn't be compiled without modifying the prj.conf. 
I'm not saying this approach is necessarily the best but that's the way it has been and something must have changed.

Reply:
There's label that builds the sample now (`CI-iot-samples-test`). It wasn't being built before, so this wasn't caught; ideally we should provide a default configuration for the sample that builds out of the box.

Reply:
OK, that makes sense.